### PR TITLE
Pong

### PR DESCRIPTION
**Pull request by fletcher97**
Added a base for pong [WIP]
Changed django setting to work in debug mode
Added websocket setup
Added a working nginx conf

### CODE REVIEW

File: compose.yaml
Comment (IsraelR1099) [inline compose.yaml:1]: Notice that :z and :Z flags labels do not work on systems that do not have SELinux, and on docker official documentation they say that those labels are ignored on services. If we keep using MacOs then these labels have no relevance

File: back/src/pong/admin.py
Comment (IsraelR1099) [inline back/src/pong/admin.py:1]: As we do not use admin app this file could be removed

**Review by davidbekic — APPROVED** [review]

**Review by davidbekic — CHANGES_REQUESTED** [review]
In compose file: nginx service still depends on front service we removed

File: compose.yaml
Comment (fletcher97) [reply compose.yaml:1]: As we'll be using a vm this should be a problem, Plus it's mainly for front dev and not necessary for prod

File: back/src/pong/admin.py
Comment (fletcher97) [reply back/src/pong/admin.py:1]: Admin has been removed

**Review by davidbekic — APPROVED** [review]

**Review by IsraelR1099 — APPROVED** [review]
it looks good for me

**Pull request merged**